developer/building/qubes-builder.md

```diff
@@ -47,20 +47,28 @@ The build system creates build environments in chroots and so no other packages
 All files created by the build system are contained within the qubes-builder directory.
 The full build requires some 25GB of free space, so keep that in mind when deciding where to place this directory.
 
+One additional useful requirement is that 'sudo root' must work without any prompt, which is default on most distros (e.g. 'sudo bash' brings you the root shell without asking for any password).
+This is important as the builder needs to switch to root and then back to user several times during the build process.
+
 The build system is configured via builder.conf file.
 You can use the setup.sh script to create and modify this file.
 Alternatively, you can copy the provided default builder.conf, and modify it as needed, e.g.:
 
 ```shell
 cp example-configs/qubes-os-master.conf builder.conf
-# edit the builder.conf file and set the following variables:
-NO_SIGN=1
 ```
 
-One additional useful requirement is that 'sudo root' must work without any prompt, which is default on most distros (e.g. 'sudo bash' brings you the root shell without asking for any password).
-This is important as the builder needs to switch to root and then back to user several times during the build process.
+The default configurations (as of `qubes-builder#master@b9f0322e4f3`)
+disables signing of generated packages, through `NO_SIGN ?= 1`.  This
+is [known not to
+work](https://github.com/QubesOS/qubes-issues/issues/6522) if you want
+to build a package that depends on another one you also build
+(eg. `linux-kernel`), and package signing must be enabled, by setting
```

Comment:
You should make it explicit that this only affects Fedora based builds, and some packages. Users wanting to modify many packages, and other templates, don't need to do this.

Reply:
AFAICT and correct me if missed something:
- the whole NO_SIGN concept today only impacts Fedora builds, by pulling `rpm-sign` and impacting `make sign`
- the `linux-yum` and `installer-qubes-os` are the only components making use of it, with code to allow unsigned repositories, but this is apparently not working today, overriden by the general setting forcing their use

right?